src/main/java/org/intellij/sdk/codesync/files/AlertsFile.java

```diff
@@ -26,7 +26,7 @@
         userEmail:
             checked_at: "2022-07-05"
             checked_for: "2022-07-05"
-            shown_at: 2022-07-05T16:30:27.210Z
+            shown_at_intellij: 2022-07-05T16:30:27.210Z
```

Comment:
I think this will be 
```
2022-07-05 16:30:27.210 UTC
```
Also, we are not using checked_at and any other main keys, we can update the comment

Reply:
Done